tests/test_users/conftest.py

```diff
@@ -8,77 +9,23 @@
 from src.apps.user.services import register_user, get_single_user, delete_all_users
 from src.apps.user.schemas import UserRegisterSchema, UserUpdateSchema, UserOutputSchema
 from src.apps.user.models import User
+from src.core.factories import UserFactory
 
-LIST_OF_USER_REGISTER_SCHEMAS = [
-    UserRegisterSchema(
-        first_name="jan",
-        last_name="kowalski",
-        email="kowaljan@mail.com",
-        birth_date=date(2020,7,12),
-        username="kowaljan2137",
-        password="kowalkowal",
-        password_repeat="kowalkowal"
-    ),
-    UserRegisterSchema(
-        first_name="karol",
-        last_name="krawczyk",
-        email="krawczyk@gmail.com",
-        birth_date=date(1965,6,18),
-        username="karkraw",
-        password="krawczyk1234",
-        password_repeat="krawczyk1234"
-    ),
-    UserRegisterSchema(  
-        first_name="norbert",
-        last_name="gierczak",
-        email="norbertgierczak@mail.com",
-        birth_date=date(1999,4,20),
-        username="jd123456",
-        password="jdjdjdjd",
-        password_repeat="jdjdjdjd")]
+DB_USER_SCHEMA = UserFactory.build(password="vgo39845n", password_repeat="vgo39845n")
 
-USER_REGISTER_DATA = {
-        "first_name": "janusz",
-        "last_name": "pawlacz",
-        "email": "janpaw@mail.com",
-        "birth_date": "2020-07-12",
-        "username": "janpaw22",
-        "password": "janusz1488",
-        "password_repeat": "janusz1488"
-        }
-
-EXISTING_USER_DATA = LIST_OF_USER_REGISTER_SCHEMAS[0]
-
-USER_UPDATE_DATA = {"email": "kowal@mailedit.com"}
-
-@pytest.fixture
-def register_existing_user_data() -> UserRegisterSchema:
-    return EXISTING_USER_DATA
-
-    
 @pytest.fixture
-def db_users(sync_session: Session):
-    delete_all_users(sync_session)
-    return [register_user(sync_session, user) for user in LIST_OF_USER_REGISTER_SCHEMAS]
-
-@pytest.fixture
-def register_data() -> dict[str, str]:
-    return USER_REGISTER_DATA
+def db_user(sync_session: Session):
+    return register_user(sync_session, DB_USER_SCHEMA)
 
 @pytest.fixture
 def login_data() -> dict[str, str]:
     return {
-        "username": LIST_OF_USER_REGISTER_SCHEMAS[0].username,
-        "password": LIST_OF_USER_REGISTER_SCHEMAS[0].password
+        "username": DB_USER_SCHEMA.username,
+        "password": DB_USER_SCHEMA.password
     }
 
 @pytest.fixture
-def get_token_header(sync_session: Session, db_users: list[UserOutputSchema]) -> dict[str, str]:
-    user_schema = get_single_user(sync_session, db_users[0].id)
+def access_token(sync_session: Session, db_user: UserOutputSchema) -> dict[str, str]:
+    user_schema = get_single_user(sync_session, db_user.id)
     access_token = AuthJWT().create_access_token(subject=user_schema.json())
```

Comment:
Gdzies mi to umknelo wczesniej, ale subjectem w JWT z reguly powinien byc scalar a nie wiekszy obiekt. Tu subjectem powinien byc albo email albo id usera, caly user nie powinien byc subjectem